src/box/sql/build.c

```diff
@@ -2367,6 +2367,85 @@ fk_constraint_change_defer_mode(struct Parse *parse_context, bool is_deferred)
 			  link)->fk_def->is_deferred = is_deferred;
 }
 
+void
+sql_drop_field_constraint(struct Parse *parse)
```

Comment:
This function looks very similar to second part of `sql_drop_core_constraint()`. I would suggest to factor the common part somehow out. Or maybe try to call the field-one from the core-one when it tries to fallback to field constraint deletion.

Reply:
I dropped this function.